Plugin~/Src/MeshSync/Include/MeshSync/msServer.h

```diff
@@ -99,6 +99,7 @@ class Server {
     
     void receivedProperty(PropertyInfoPtr prop);
     void syncRequested();
+    void scriptCallbackRequested();
```

Comment:
Could this be renamed to userScriptCallbackRequested() ?

Reply:
Done